docs/src/pages/components/alert/TransitionAlerts.tsx

```diff
@@ -1,30 +1,19 @@
 import * as React from 'react';
-import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import Alert from '@material-ui/core/Alert';
 import IconButton from '@material-ui/core/IconButton';
 import Collapse from '@material-ui/core/Collapse';
 import Button from '@material-ui/core/Button';
 import CloseIcon from '@material-ui/icons/Close';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      width: '100%',
-      '& > * + *': {
-        marginTop: theme.spacing(2),
-      },
-    },
-  }),
-);
-
 export default function TransitionAlerts() {
-  const classes = useStyles();
   const [open, setOpen] = React.useState(true);
 
   return (
-    <div className={classes.root}>
+    <Box sx={{ width: '100%' }}>
       <Collapse in={open}>
         <Alert
+          sx={{ marginBottom: (theme) => theme.spacing(2) }}
```

Comment:
I took this approach because `<Button />` would overwrite the margin if the selector  `& > * + *` was used.

Reply:
what about sx={{mb: 2}} ?